react/src/pages/aws/InfrastructurePage.js

```diff
@@ -326,15 +386,16 @@ const Subnet = (props) => {
     const toggleTags = () => toggleShow("showTags");
     return <>
         <div className={"box margin" + (subnet?.type === "private" ? " darken" : " lighten")} style={{width:"100%"}}>
-            <div style={{borderBottom:"1px solid var(--box-fg)",paddingBottom:"2pt",marginBottom:"4pt"}}>
+            <div>
                 <b>Subnet</b>: <b style={{color:"black"}}>{subnet?.name}</b>
                 <ExternalLink
                     href={`https://us-east-1.console.aws.amazon.com/vpc/home?region=us-east-1#SubnetDetails:subnetId=${subnet?.id}`}
                     bold={true}
                     style={{marginLeft:"6pt"}} />
                 <small style={{float:"right"}}>
```

Comment:
why is this floating? just curious.

actually, while i'm asking, why is it a small and not a span with point size info? it's not a change to make locally, but I was taught that things like small and whatnot are ways to the characteristics of something but they are not building blocks of layout (span for horizontal and div for vertical, mostly, though also p, ul, etc. in more specialized ways). while you can make anything float, it just looks odd to my eye. this is not a request for change.